### ISSUE TITLE
Add Build / installation instructions to README

### ISSUE THREAD

**Issue by Bios-Marcel** [tobiasbaum/reviewtool#73]
Hey,

I'd like to try this out, but there's no installation instructions as far as I can see. This would be helpful. I am assuming it's juts `mvn package` and dropping the JAR file into the plugins folder of Eclipse?

**Comment by tobiasbaum** [comment]
Good point. I added a bit of description and also uploaded the most recent release as a zipped update site. Sadly, there is not much detailed documentation at the moment, but let me know if you have problems and I'll be glad to help.
